tutorials/fundamentals/check-nans.md

```diff
@@ -0,0 +1,11 @@
+When dealing with NaNs in your projects, it is important to understand what Nans are and how they work. NaN is a non-writable, non-configurable, non-enumerable property of the global object. You should avoid overriding it an instead error handle it when the situation arises in your project development. To do that however, you must check whether the value is a NaN. A interesting thing about Nans is that `NaN === NaN` and `Number.NaN === NaN` will return false as in the compiler's eyes they are not the same thing. Instead you can do the following:
```

Comment:
No compiler in JS, so this should be something more like "An interesting thing about `NaN` is that `NaN !== NaN` and `Number.NaN !== NaN`